src/metadata/services/tags.clj

```diff
@@ -127,7 +164,7 @@
      limit - The `limit` query parameter. It should be a positive integer."
   [user contains limit]
   (let [matches (db/get-tags-by-value user (str "%" contains "%") limit)]
-    {:tags (map #(dissoc % :owner_id) matches)}))
+    {:tags matches}))
```

Comment:
The owner ID isn't actually returned by the query, so I just deleted the code that was being used to remove it.